common/kvstore/src/test/java/org/apache/spark/util/kvstore/LevelDBSuite.java

```diff
@@ -276,6 +277,41 @@ public void testNegativeIndexValues() throws Exception {
     assertEquals(expected, results);
   }
 
+  @Test
+  public void testCloseLevelDBIterator() throws Exception {
```

Comment:
I haven't tested but it might not fail on Linux/MacOS even without the patch. That's OK, if you've tested with Windows and make sure this test fails without the patch. Could you please confirm? (I don't have Windows env. for development.)

Reply:
yes, this is tested on windows, without this patch, assert in line 312 would fail.